composer/cli/launcher.py

```diff
@@ -466,12 +447,14 @@ def main():
     log.setLevel(logging.INFO if args.verbose else logging.WARN)
 
     processes = {}
-
     log_tmpdir = tempfile.TemporaryDirectory()
-    if args.stdout is None:
-        args.stdout = f'{log_tmpdir.name}/rank{{rank}}.stdout.txt'
-    if args.stderr is None:
-        args.stderr = f'{log_tmpdir.name}/rank{{rank}}.stderr.txt'
+    log_file_format = f'{log_tmpdir.name}/rank{{rank}}.txt'
```

Comment:
I'm thinking we should preserve the option of the old behavior somehow since this is an api breaking change. @mvpatel2000 wdyt?